8_distributed_training.ipynb

```diff
@@ -72,13 +72,22 @@
     "\n",
     "2. Click *Display token*.\n",
     "\n",
-    "3. Below *Log in with this token*, copy the line that has the following format:\n",
-    "\n",
-    "    `oc login --token=sha256~XXXX --server=https://XXXX`\n",
-    "\n",
-    "    You can run the `oc login` command the same way that you run any terminal command. You can designate a cell as `%%bash` or you can paste a command after `!`.\n",
-    "\n",
-    "4. Paste your `oc login --token=sha256~XXXX --server=https://XXXX` command in the following cell designated with `%%bash` and run it.\n"
+    "3. Below *Log in with this token*, take note of the parameters for token and server.  \n",
+    "    ```\n",
+    "    oc login --token=sha256~LongString --server=https://api.your-cluster.domain.com:6443\n",
+    "    ```    \n",
+    "    - token: `sha256~LongString`\n",
+    "    - server: `https://api.your-cluster.domain.com:6443`\n",
+    "    \n",
+    "4. In the TokenAuthentication object, replace the token and server with the values you copied from the OpenShift web console.\n",
```

Comment:
4. In the following code cell, in the the TokenAuthentication object, replace the token and server values with the values that you noted in Step 3.
For example: